home-application/src/main/resources/config/application.properties

```diff
@@ -3,3 +3,4 @@ management.endpoints.web.exposure.include=health
 management.endpoint.health.show-details=always
 server.port=${PORT:8080}
 spring.jersey.type=filter
+spring.jpa.hibernate.ddl-auto=validate
```

Comment:
Add new line in the end of the file, please.

Reply:
sure